package.json

```diff
@@ -32,11 +32,14 @@
     "@angular/platform-browser-dynamic": "4.0.1",
     "@angular/router": "4.0.1",
     "bulma": "^0.4.0",
+    "electron-simple-updater": "^1.2.0",
     "hterm": "0.0.1",
+    "httpreq": "^0.4.23",
     "ng2-slimscroll": "^1.3.0",
     "node-pty": "^0.6.3",
     "reflect-metadata": "0.1.10",
     "rxjs": "5.3.0",
+    "semver": "^5.3.0",
```

Comment:
dependencies work in the way if one package requires it it gets installed automatically, you don't need to specify it again. check [here](https://github.com/megahertz/electron-simple-updater/blob/master/package.json#L55-L56). so, `npm install electron-simple-updater --save` should be enough, please remove `semver` and `httpreq`.